app/services/api/v3/participant_declarations_query.rb

```diff
@@ -51,9 +48,14 @@ def participant_declarations_from(paginated_join)
               :cpd_lead_provider,
             )
             .from("(#{sub_query.to_sql}) as participant_declarations")
+            .order(:created_at)
             .distinct
 
-        scope.order(:created_at)
+        preloader = ActiveRecord::Associations::Preloader.new
+        preloader.preload(scope.select { |p| p.type == "ParticipantDeclaration::NPQ" }, :outcomes)
+        preloader.preload(scope.select { |p| p.type == "ParticipantDeclaration::NPQ" }, participant_profile: :npq_application)
```

Comment:
I'm sure there's room for improvement here but as it's working let's go with it for now